### DISCUSSION TITLE
Airplay device not showing (portainer)

### DISCUSSION THREAD

**Discussion by monstruwaken** [mikebrady/shairport-sync#1721] [Q&A]
![IMG_8523](https://github.com/mikebrady/shairport-sync/assets/129773246/1e678dd2-b0fd-492a-8a48-29981c3cc8ff)
Docker installed and running in Portainer but is not showing in airplay devices. Is there something that I am missing?

**Comment by mikebrady** [comment]
Thanks for the post, but this is really more suitable as an Issue. If you could submit it as an Issue we can take it from there.
